packages/cmf/src/store.js

```diff
@@ -40,6 +41,16 @@ function setRouterMiddleware(middleware) {
 	defaultRouterOverwrite = true;
 }
 
+/**
+ * setHttpMiddleware overwrites the default router middleware
+ *
+ * @param middleware a router middleware
+ */
+function setHttpMiddleware(middleware) {
+	middlewares.push(middleware);
```

Comment:
this original order is not kept.
Don't know if this has an impact but it may have one.